/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loggers/wandb.py:396: UserWarning: There is a wandb run already in progress and newly created instances of `WandbLogger` will reuse this run. If this is not desired, call `wandb.finish()` before instantiating `WandbLogger`.
  rank_zero_warn(
GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
  | Name         | Type      | Params
-------------------------------------------
0 | bert         | BertModel | 89.1 M
1 | hidden_layer | Linear    | 49.2 K
2 | layer        | Linear    | 65
3 | criterion    | BCELoss   | 0
-------------------------------------------
7.1 M     Trainable params
82.0 M    Non-trainable params
89.2 M    Total params
356.650   Total estimated model params size (MB)


Epoch 0:   0% 0/4 [00:00<?, ?it/s]
/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/fit_loop.py:280: PossibleUserWarning: The number of training batches (4) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.



Epoch 0: 100% 4/4 [03:27<00:00, 51.80s/it, v_num=31nn, train/loss=0.708]

        [0.4591],
        [0.4630],
        [0.4541],
        [0.4920],
        [0.4627],
        [0.4502],
        [0.4709],
        [0.4545],
        [0.4643],
        [0.4612],
        [0.4451],
        [0.4750],
        [0.4558],
        [0.4331],
        [0.4726]], grad_fn=<SigmoidBackward0>), tensor([[0.4808],
        [0.4757],
        [0.4698],
        [0.4628],
        [0.4649],
        [0.4650],
        [0.4697],
        [0.4685],
        [0.5019],
        [0.4655],
        [0.4714],
        [0.4921],
        [0.4683],
        [0.4758],
        [0.4790],
        [0.4870]], grad_fn=<SigmoidBackward0>), tensor([[0.4878],
        [0.4761],
        [0.4661],
        [0.4747],
        [0.4812],
        [0.4911],
        [0.4593],
        [0.4975],
        [0.4831],
        [0.4899],
        [0.4868],
        [0.4794],
        [0.4775],
        [0.4770],
        [0.4758],
        [0.4799]], grad_fn=<SigmoidBackward0>), tensor([[0.4875],
        [0.4853],
        [0.4842],
        [0.4941],
        [0.5001],
        [0.4910],
        [0.5077],
        [0.4831],
        [0.5024],
        [0.4872],
        [0.4770],
        [0.4897]], grad_fn=<SigmoidBackward0>)]
[tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]), tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]), tensor([1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]), tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 1, 1])]
Error executing job with overrides: []
Traceback (most recent call last):
  File "/content/drive/MyDrive/murata_labo_exp/murata_labo_exp_src/exp3/main.py", line 390, in main
    trainer.fit(model, data_module)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 529, in fit
    call._call_and_handle_interrupt(
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py", line 42, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 568, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 973, in _run
    results = self._run_stage()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/trainer.py", line 1016, in _run_stage
    self.fit_loop.run()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/fit_loop.py", line 202, in run
    self.on_advance_end()
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/loops/fit_loop.py", line 368, in on_advance_end
    call._call_lightning_module_hook(trainer, "on_train_epoch_end")
  File "/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py", line 144, in _call_lightning_module_hook
    output = fn(*args, **kwargs)
  File "/content/drive/MyDrive/murata_labo_exp/murata_labo_exp_src/exp3/main.py", line 213, in on_train_epoch_end
    epoch_preds = torch.stack(self.train_step_outputs_preds)
RuntimeError: stack expects each tensor to be equal size, but got [16, 1] at entry 0 and [12, 1] at entry 3
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.